libs/core/src/lib/select/select.component.ts

```diff
@@ -119,6 +126,9 @@ export class SelectComponent implements OnChanges, AfterContentInit, OnDestroy,
     /** Subject triggered when the component is destroyed. */
     private readonly destroy$: Subject<void> = new Subject<void>();
 
+    @Input() loading: boolean = false;
```

Comment:
add doc comments?

Reply:
also might want to group this input with all the other inputs in this file, and follow the whole 2-line input thing, just for cleanliness

Reply:
Were waiting for designers for further implimentation